docs/setuptools.txt

```diff
@@ -239,179 +239,6 @@ pre- or post-release tags.  See the following sections for more details:
 * The `egg_info`_ command
 
 
-New and Changed ``setup()`` Keywords
```

Comment:
Can we leave this header in place with link to the new keywords document? People are likely linking directly to some headers on this page, so I like to leave a stub in place pointing to the new location, at least for some time.

Reply:
Done.